test/viewmodels/IndexPageHelperSpec.scala

```diff
@@ -1245,6 +1245,8 @@ class IndexPageHelperSpec extends SpecBase {
                 penaltyStatus = LPPPenaltyStatusEnum.Accruing,
                 penaltyAmountPaid = Some(0),
                 penaltyAmountOutstanding = Some(144.21),
+                penaltyAmountPosted = 0,
```

Comment:
penaltyAmountOutstanding would be None and penaltyAmountAccruing would be 144.21